frontend/src/pages/destructionlist/detail/DestructionListDetail.tsx

```diff
@@ -169,25 +149,36 @@ export async function destructionListUpdateAction({
  * React Router loader.
  */
 export const destructionListDetailLoader = loginRequired(
-  async ({ request, params }: ActionFunctionArgs) => {
-    if (typeof params.uuid === "undefined") return {};
+  async ({
+    request,
+    params,
+  }: ActionFunctionArgs): Promise<DestructionListDetailContext> => {
+    console.log("loader", request, params);
+
+    const storageKey = `destruction-list-detail-${params.uuid}`;
+    const searchParamsZakenEndpoint: Record<string, string> = {
+      not_in_destruction_list_except: String(params.uuid),
+    };
+    const searchParams = new URL(request.url).searchParams;
+    Object.keys(searchParamsZakenEndpoint).forEach((key) =>
+      searchParams.set(key, searchParamsZakenEndpoint[key]),
+    );
 
+    // Get reviewers, zaken and zaaktypen.
     const promises = [
-      getDestructionList(params.uuid),
-      getZakenData(request, {
-        not_in_destruction_list_except: String(params.uuid),
-      }),
+      getDestructionList(params.uuid as string),
       listReviewers(),
+      listZaken(searchParams),
     ];
-
-    const [destructionList, zakenListData, availableReviewers] =
-      await Promise.all(promises);
+    const [destructionList, reviewers, allZaken] = (await Promise.all(
+      promises,
+    )) as [DestructionList, User[], PaginatedZaken];
 
     return {
-      availableReviewers,
+      allZaken,
```

Comment:
Based on the changes here: https://github.com/maykinmedia/open-archiefbeheer/pull/116
The destruction list won't contain the zaken nested anymore, so a call to get the zaken in the destruction list would also be needed.